trade-finance-manager-ui/server/controllers/case/cancellation/bank-request-date.controller.post.test.ts

```diff
@@ -151,6 +152,37 @@ describe('postBankRequestDate', () => {
           day,
           month,
           year,
+          previousPage: defaultBackUrl,
+        });
+      });
+
+      it('renders the page with the back URL as the check details page when "change" is passed in as a query parameter', async () => {
+        // Arrange
+        const { req, res } = createMocks<PostBankRequestDateRequest>({
+          params: { _id: dealId },
+          query: { status: 'change' },
+          session: {
+            user: mockUser,
+            userToken: 'a user token',
```

Comment:
If Mark's PR [here](https://github.com/UK-Export-Finance/dtfs2/pull/3661) gets merged before this, can we update such instances to use `aRequestSession` please 🙏